### PR TITLE
#175815411 Bug: solving eslint bug 

### PR DESCRIPTION
**Pull request by tigthor**
### What does this PR do?
- solving eslint issues
- Adding eslint scripts
### Description of Task to be completed?
N/A
### How should this be manually tested?
N/A
### Any background context you want to provide?
N/A
### What are the relevant pivotal tracker stories?
N/A 
### Screenshot (if appropriate) 
N/A

[finish #175494029]

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,8 +4,10 @@
   "description": "This is the visit africa backend",
   "main": "src/server.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start-dev": "nodemon --exec babel-node src/server.js"
+    "test": "/node_modules/.bin/mocha --require @babel/register src/test/*.js --exit",
+    "startDev": "nodemon --exec babel-node src/server.js",
```
Comment (erickyvand) [inline package.json]: why did you change these lines as these were nothing to do with eslint?

File: src/tests/test.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  it('simple message to be displayed', (done) => {
+    chai
+      .request(app)
+      .get('/')
+      .end((err, res) => {
+        expect(res.status).to.be.equal(200);
+        done();
+      });
+  });
+});
```
Comment (erickyvand) [inline src/tests/test.js]: Any reason why these codes are in `fix-eslint`?

File: package.json [hunk 3]
```diff
@@ -35,10 +31,21 @@
     "eslint-plugin-mocha": "^8.0.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-standard": "^4.1.0",
-    "nodemon": "^2.0.6"
+    "l": "^0.6.0",
```
Comment (erickyvand) [inline package.json]: What does this package stand for?

**Review by erickyvand — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (tigthor) [reply package.json]: It comes with chai

File: .hound.yml [hunk 4]
```diff
@@ -2,4 +2,4 @@
 
 eslint:
   enabled: true
-  config_file: ".eslintrc.json"
+  config_file: .eslintrc
```
Comment (erickyvand) [inline .hound.yml]: You tried to correct this line but you should not remove the extension, it should be like this `config_file: .eslintrc.json`

File: package.json [hunk 5]
```diff
@@ -4,8 +4,10 @@
   "description": "This is the visit africa backend",
   "main": "src/server.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start-dev": "nodemon --exec babel-node src/server.js"
+    "test": "./node_modules/.bin/mocha --require @babel/register src/test/*.js --exit",
```
Comment (erickyvand) [inline package.json]: I keep saying that you shouldn't change this line as it will be changed on it own PR. Leave it as it is.

File: package.json (hunk 3 above)
Comment (erickyvand) [reply package.json]: No need for that package

**Review by erickyvand — CHANGES_REQUESTED** [review]

File: .hound.yml (hunk 4 above)
Comment (tigthor) [reply .hound.yml]: Yeah I am looking on that

File: package.json (hunk 5 above)
Comment (tigthor) [reply package.json]: Yeah No problem I am letting it as it is

**Comment by tigthor** [comment]
@erickyvand  Please review now

**Review by erickyvand — APPROVED** [review]

**Review by Niyonsengaeric — APPROVED** [review]
